src/gwc/src/main/java/org/geoserver/gwc/dispatch/GwcServiceProxy.java

```diff
@@ -134,5 +135,15 @@ public void write(byte[] b, int off, int len) throws IOException {
         public byte[] getBytes() {
             return outputStream.toByteArray();
         }
+
+        @Override
+        public boolean isReady() {
+            return false;
```

Comment:
Hmm...  I believe this should return true, since we can actually write without blocking:

https://docs.oracle.com/javaee/7/api/javax/servlet/ServletOutputStream.html#isReady--

Probably not used at all right now, but does not hurt to code it correctly.

Reply:
Done.